From 5f3e2e6b006301c99e9f859b9410676b94017600 Mon Sep 17 00:00:00 2001
From: Lili Li <lili.li@intel.com>
Date: Tue, 30 Nov 2021 17:36:02 +0800
Subject: [PATCH] ice: fix: Fix ice_pin_desc_e10t compile error
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Compile error:
drivers/net/ethernet/intel/ice/ice_ptp.c:11:2: error: braces around scalar initializer [-Werror]
   11 |  { "GNSS",  GNSS, PTP_PF_EXTTS, 0, { 0, } },
      |  ^
drivers/net/ethernet/intel/ice/ice_ptp.c:11:2: note: (near initialization for ‘ice_pin_desc_e810t[0].flags’)
drivers/net/ethernet/intel/ice/ice_ptp.c:12:2: error: braces around scalar initializer [-Werror]
   12 |  { "SMA1",  SMA1, PTP_PF_NONE, 1, { 0, } },
      |  ^
drivers/net/ethernet/intel/ice/ice_ptp.c:12:2: note: (near initialization for ‘ice_pin_desc_e810t[1].flags’)
drivers/net/ethernet/intel/ice/ice_ptp.c:13:2: error: braces around scalar initializer [-Werror]
   13 |  { "U.FL1", UFL1, PTP_PF_NONE, 1, { 0, } },
      |  ^
drivers/net/ethernet/intel/ice/ice_ptp.c:13:2: note: (near initialization for ‘ice_pin_desc_e810t[2].flags’)
drivers/net/ethernet/intel/ice/ice_ptp.c:14:2: error: braces around scalar initializer [-Werror]
   14 |  { "SMA2",  SMA2, PTP_PF_NONE, 2, { 0, } },
      |  ^
drivers/net/ethernet/intel/ice/ice_ptp.c:14:2: note: (near initialization for ‘ice_pin_desc_e810t[3].flags’)
drivers/net/ethernet/intel/ice/ice_ptp.c:15:2: error: braces around scalar initializer [-Werror]
   15 |  { "U.FL2", UFL2, PTP_PF_NONE, 2, { 0, } },
      |  ^
drivers/net/ethernet/intel/ice/ice_ptp.c:15:2: note: (near initialization for ‘ice_pin_desc_e810t[4].flags’)

Fixes: 325b2064d00a ("ice: Implement support for SMA and U.FL on E810-T")
Signed-off-by: Lili Li <lili.li@intel.com>
---
 drivers/net/ethernet/intel/ice/ice_ptp.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/net/ethernet/intel/ice/ice_ptp.c b/drivers/net/ethernet/intel/ice/ice_ptp.c
index bf7247c6f58e..8a892320f2b0 100644
--- a/drivers/net/ethernet/intel/ice/ice_ptp.c
+++ b/drivers/net/ethernet/intel/ice/ice_ptp.c
@@ -8,11 +8,11 @@
 
 static const struct ptp_pin_desc ice_pin_desc_e810t[] = {
 	/* name    idx   func         chan */
-	{ "GNSS",  GNSS, PTP_PF_EXTTS, 0, { 0, } },
-	{ "SMA1",  SMA1, PTP_PF_NONE, 1, { 0, } },
-	{ "U.FL1", UFL1, PTP_PF_NONE, 1, { 0, } },
-	{ "SMA2",  SMA2, PTP_PF_NONE, 2, { 0, } },
-	{ "U.FL2", UFL2, PTP_PF_NONE, 2, { 0, } },
+	{ "GNSS",  GNSS, PTP_PF_EXTTS, 0, 0, { 0, } },
+	{ "SMA1",  SMA1, PTP_PF_NONE, 1, 0, { 0, } },
+	{ "U.FL1", UFL1, PTP_PF_NONE, 1, 0, { 0, } },
+	{ "SMA2",  SMA2, PTP_PF_NONE, 2, 0, { 0, } },
+	{ "U.FL2", UFL2, PTP_PF_NONE, 2, 0, { 0, } },
 };
 
 /**
-- 
2.32.0

